modules/common/annotations/network.go

```diff
@@ -54,3 +55,44 @@ func GetNADAnnotation(namespace string, nads []string) (map[string]string, error
 
 	return map[string]string{NetworkAttachmentAnnot: string(networks)}, nil
 }
+
+// GetAnnotation - returns an annotation value based on the key passed as input
+// This function also returns true of false based on the fact that a value associated
+// to the key passed as input exists. Returning a boolean is useful to allow wrapping
+// this function in calls like `value, ok := GetAnnotation(ann, key); ok {` and
+// take actions based on the existance of the entry in the annotation map[string]string
+func GetAnnotation(
+	ann map[string]string,
+	key string,
+) (string, bool) {
+	if value, ok := ann[key]; ok {
+		return value, ok
+	}
+	return "", false
+}
```

Comment:
do we need this func? isn't it the same as `return value, exists := ann[key]`

Reply:
I added this because someone might want to not use this and not the one that checks the boolean and get an additional `exists` information through a bool.
If you think we don't need it we can remove it, it is just a utility that might be helpful to be wrapped and build something else around it.

Reply:
Removed